### PR TITLE
Support multiple Carousel indicators lists

### PR DESCRIPTION
**Pull request by MikhailRoot**
Carousel now supports multiple carousel-indicators lists to switch .active class on all of them simultaneously.

### CODE REVIEW

**Comment by Johann-S** [comment]
Maybe you can add a preview of this feature and fix CI issues please

**Comment by MikhailRoot** [comment]
@Johann-S   Created a demo of this feature here: https://codepen.io/MikhailRoot/full/VbEPwy/ 
 The idea and need was to support multiple carousel slides indicators.
Before my edits it just took the first set of indicators and removed .active class from any other.
Also updated code to use jQuery each iterator instead of for loop.

File: js/carousel.js [hunk 1]
```diff
@@ -137,8 +137,11 @@
 
     if (this.$indicators.length) {
       this.$indicators.find('.active').removeClass('active')
-      var $nextIndicator = $(this.$indicators.children()[this.getItemIndex($next)])
-      $nextIndicator && $nextIndicator.addClass('active')
+      var nextItemIndex = this.getItemIndex($next)
+      $(this.$indicators).each(function(index,indicator){
+        var $nextIndicator = $( $(indicator).children()[nextItemIndex])
```
Comment (vsn4ik) [inline js/carousel.js]: `$( $(indicator)...)` -> `$($(indicator)...)`

**Review by vsn4ik — CHANGES_REQUESTED** [review]

**Comment by MikhailRoot** [comment]
Travis CI for some reason fails with "Your Gemfile.lock is corrupt. The following gem is missing from the DEPENDENCIES"

File: js/carousel.js (hunk 1 above)
Comment (MikhailRoot) [inline js/carousel.js]: Done :)

File: js/carousel.js [hunk 2]
```diff
@@ -137,8 +137,11 @@
 
     if (this.$indicators.length) {
       this.$indicators.find('.active').removeClass('active')
-      var $nextIndicator = $(this.$indicators.children()[this.getItemIndex($next)])
-      $nextIndicator && $nextIndicator.addClass('active')
+      var nextItemIndex = this.getItemIndex($next)
+      $(this.$indicators).each(function(index,indicator){
```
Comment (vsn4ik) [inline js/carousel.js]: `(index,indicator)` -> `(index, indicator)`&

**Comment by Johann-S** [comment]
Bootstrap 3 is essentially in maintenance mode as we focus on working towards a stable v4. As such, we're only accepting changes to v3's code on a case-by-case basis, usually only for critical bug fixes or docs improvements.

If you want you can submit your PR for our `v4` release

**Pull request closed without merging**